mlflow/sklearn/__init__.py

```diff
@@ -817,31 +817,13 @@ def _log_posttraining_metadata(estimator, *args, **kwargs):
                      `fit()`, `fit_transform()`, ...).
         :param kwargs: The keyword arguments passed to the scikit-learn training routine.
         """
-        if hasattr(estimator, "score"):
-            try:
-                score_args = _get_args_for_score(estimator.score, estimator.fit, args, kwargs)
-                training_score = estimator.score(*score_args)
-            except Exception as e:
-                msg = (
-                    estimator.score.__qualname__
-                    + " failed. The 'training_score' metric will not be recorded. Scoring error: "
-                    + str(e)
-                )
-                _logger.warning(msg)
-            else:
-                try_mlflow_log(mlflow.log_metric, "training_score", training_score)
 
         # log common metrics and artifacts for estimators (classifier, regressor)
-        _log_specialized_estimator_content(estimator, mlflow.active_run().info.run_id, args, kwargs)
 
         def get_input_example():
             # Fetch an input example using the first several rows of the array-like
             # training data supplied to the training routine (e.g., `fit()`)
-            fit_arg_names = _get_arg_names(estimator.fit)
-            X_var_name, y_var_name = fit_arg_names[:2]
-            input_example = _get_Xy(args, kwargs, X_var_name, y_var_name)[0][
-                :INPUT_EXAMPLE_SAMPLE_ROWS
-            ]
+            input_example = X[:INPUT_EXAMPLE_SAMPLE_ROWS]
```

Comment:
A bit strange to reference a global var that is not yet defined. Move it after L838?

Reply:
My bad. Moved it later.